src/Root.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="people" element={<PeoplePage />}>
+            <Route path=":personSlug?" element={<PeoplePage />} />
```

Comment:
The `PeoplePage` component is used for both the `people` path and the `:personSlug?` path. If `PeoplePage` is intended to handle both cases, this is fine. Otherwise, consider whether a different component should be used for the `:personSlug?` path to avoid redundancy.